source/extensions/filters/network/zookeeper_proxy/decoder.cc

```diff
@@ -473,6 +473,13 @@ absl::Status DecoderImpl::parseCreateRequest(Buffer::Instance& data, uint64_t& o
                                                                flag_data.status().message());
 
   const CreateFlags flags = static_cast<CreateFlags>(flag_data.value());
+
+  if (opcode == OpCodes::CreateTtl) {
+    absl::StatusOr<int64_t> ttl = helper_.peekInt64(data, offset);
```

Comment:
`ttl` seems not being used. is that intentional?

Reply:
Yeah, that's intentional. The main purpose of the change is to make sure the buffer offset is correct.